testwheel.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+
+import sys
+sys.path.insert(0, 'grudge-2015.1-py3-none-any.whl')
+sys.path.insert(0, 'dagrt-2019.4-py3-none-any.whl')
+sys.path.insert(0, 'leap-2019.5-py3-none-any.whl')
+sys.path.insert(0, 'loo.py-2019.1-py3-none-any.whl')
+sys.path.insert(0, 'meshmode-2016.1-py3-none-any.whl')
```

Comment:
I know it is the Matlab(tm) way, but can you look for a way of sticking these into a virtualenv?
```
pip install something.whl
```
might suffice... and it might not even unpack much.

Reply:
Ok, I can try that. I was afraid that this might unpack lots of stuff and cause the same FS issues we saw before.

Reply:
You should be able to check simply by poking around in the virtualenv folder.